forte/api/state_info_api.cc

```diff
@@ -27,6 +27,7 @@
  */
 
 #include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
```

Comment:
I'm assuming you get compiler issues if you remove this?

Reply:
This is needed. If absent a runtime error is emitted complaining about conversion of data (IIRC).